features/dashboard.feature

```diff
@@ -1,28 +1,12 @@
 Feature: Dashboard Interactions
 
-
-#  Scenario: create a named dashboard - capybara failing
```

Comment:
do we want this test? should it fail, or pass? Can we make it officially pending if we can't make it work right now, rather than commenting it?